docs/validations/validations-rules-reference.mdx

```diff
@@ -0,0 +1,24 @@
+---
+sidebar_label: 'Rules Reference'
+---
+
+# Rules Reference
+
+Rules can be asserted against a gameplay recording to ensure that the recording meets specific criteria. We provide a set of rule types that can be used to construct assertions based on various performance metrics.
```

Comment:
I think we're missing an answer to the question "How is this different than a validation scenario?". Probably want to reword this to include that rules are applied to every tick in the gameplay session and act more as monitoring than a validation.

We'll also want to clarify somewhere that rules produce warnings, not errors.

Reply:
good call, added a bit about this